resources/favEvents/favEvents.route.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+
+const router = express.Router();
+
+const favEventsController = require('./favEvents.controller');
+
+router.get('/', async (req, res) => {
+  try {
+    const { u } = req.query;
+    if (typeof u === 'undefined') {
+      res.status(400).json('Please supply valid id in the query');
+    } else {
+      const favEventsId = await favEventsController.findFav(u);
+      if (!favEventsId) {
+        throw new Error('Favorite events could not be retrieved at this time. Please try again later');
```

Comment:
Throwing new error doesn't tell the user anything about their error, this should have a response of the error or what happen

Reply:
Alright mate